src/components/NavContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Text, View } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+import HomeScreen from '../screens/HomeScreen';
+import TaskScreen from '../screens/TaskScreen';
+import BadgeScreen from '../screens/BadgeScreen';
+import ProfileScreen from '../screens/ProfileScreen';
+
+const Tab = createBottomTabNavigator();
+
+export default function NavFlow() {
+  return (
+    <NavigationContainer>
+      <Tab.Navigator
+        screenOptions={({ route }) => ({
+          tabBarIcon: ({ color, size }) => {
+            let iconName;
+
+            if (route.name === 'Home') {
+              iconName = 'ios-home';
+            } else if (route.name === 'Tasks') {
+              iconName = 'ios-list';
+            } else if (route.name === 'Progress') {
+              iconName = 'ios-clipboard';
+            } else if (route.name === 'Me') {
+              iconName = 'ios-person';
+            }
+
+            return <Ionicons name={iconName} size={size} color={color} />;
+          },
+        })}
+        tabBarOptions={{
+          activeTintColor: '#E96661',
+          inactiveTintColor: 'gray',
+        }}
+      >
+        <Tab.Screen name="Home" component={HomeScreen} />
+        <Tab.Screen name="Tasks" component={TaskScreen} />
+        <Tab.Screen name="Progress" component={BadgeScreen} />
+        <Tab.Screen name="Me" component={ProfileScreen} />
```

Comment:
We can probably have a JS object mapping screen name to screen component, then do obj.keys().map() here, if we ever have more screens to add, or if the amount of screens becomes dynamic in the future (i.e. we have different types of users who see different types of screens). Not necessary if we end up only using 4 screens here, but just something good to have in the future.

Reply:
For the scope of our work on YCW, we'll never have more than 4 screens, so I think this isn't essential for this PR. But it would definitely be smart in case YCW wants to expand on our work. 

@hanakims Let's leave this change for last, so if we have time we can include it in this PR.